### PR TITLE
magento/devdocs#: Minor grammar fixes

### PR DESCRIPTION
**Pull request by DashukVolodymyr**
## Purpose of this pull request

This pull request (PR) provides minor grammar fixes to the existing documentation.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/bk-frontend-dev-guide.html
- https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.html 
- https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/css-topics/css-overview.html
- https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/layout-types.html
- https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/responsive-web-design/rwd-breakpoints.html
- https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/theme-best-practice.html
- https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/themes/product-video.html
- https://devdocs.magento.com/guides/v2.4/frontend-dev-guide/bk-frontend-dev-guide.html

## Links to Magento source code

-  https://developer.mozilla.org/en-US/docs/Web/CSS

### CODE REVIEW

File: src/guides/v2.3/frontend-dev-guide/bk-frontend-dev-guide.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Frontend developers can use this guide to create custom themes to tailor the Mag
 
 You can apply these levels of customization to your site, where the levels require different development skills:
 
-*  You can make relatively simple changes to your site using cascading style sheets (CSS) to change colors and the look-and-feel of various interface components, replacing images, and other relatively cosmetic changes. For more advanced styling, familiarity with LESS (Leaner Style Sheets) and XML (Extensible Markup Language) is helpful.
+*  You can make relatively simple changes to your site using Cascading Style Sheets (CSS) to change colors, and the look-and-feel of various interface components, replacing images, and other relatively cosmetic changes. For more advanced styling, familiarity with LESS (Leaner Style Sheets) and XML (Extensible Markup Language) is helpful.
```
Comment (BarnyShergold) [inline src/guides/v2.3/frontend-dev-guide/bk-frontend-dev-guide.md:19]: In English we don't put a comma before "and" unless the preceding part of the sentence is a list.
```suggestion
*  You can make relatively simple changes to your site using Cascading Style Sheets (CSS) to change colors and the look-and-feel of various interface components, replacing images, and other relatively cosmetic changes. For more advanced styling, familiarity with LESS (Leaner Style Sheets) and XML (Extensible Markup Language) is helpful.
```

File: src/guides/v2.3/frontend-dev-guide/bk-frontend-dev-guide.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ You can apply these levels of customization to your site, where the levels requi
 
 ## Frontend development prerequisites {#fedg-prereqs}
 
-To implement what is discussed in this guide, you need a working Magento installation and the following browser versions installed on your device:
+To implement what is discussed in this guide, you need a working Magento installation, and the following browser versions installed on your device:
```
Comment (BarnyShergold) [inline src/guides/v2.3/frontend-dev-guide/bk-frontend-dev-guide.md:47]: As above
```suggestion
To implement what is discussed in this guide, you need a working Magento installation and the following browser versions installed on your device:
```

File: src/guides/v2.3/frontend-dev-guide/layouts/layout-types.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ This article gives a comprehensive description of each layout file type.
 
 ## Page layout {#layout-types-page}
 
-Page layout declares the wireframe of a page inside the `<body>` section, for example one-column layout or two-column layout.
+Page layout declares the wireframe of a page inside the `<body>` section, for example, one-column layout or two-column layout.
```
Comment (BarnyShergold) [inline src/guides/v2.3/frontend-dev-guide/layouts/layout-types.md:23]: Normally use of 2 commas like this means the section between the commas can be "removed" and the sentence still makes sense. Not the case here, so full stop looks better
```suggestion
Page layout declares the wireframe of a page inside the `<body>` section. For example, one-column layout or two-column layout.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Some suggestions.

File: src/guides/v2.3/frontend-dev-guide/bk-frontend-dev-guide.md (hunk 2 above)
Comment (DashukVolodymyr) [reply src/guides/v2.3/frontend-dev-guide/bk-frontend-dev-guide.md:47]: Hi @BarnyShergold 
Thanks for your notes!
Should I revert my changes here?

File: src/guides/v2.3/frontend-dev-guide/layouts/layout-types.md (hunk 3 above)
Comment (DashukVolodymyr) [reply src/guides/v2.3/frontend-dev-guide/layouts/layout-types.md:23]: Thanks for your notes!
Should I revert my changes here?

Comment (BarnyShergold) [reply src/guides/v2.3/frontend-dev-guide/layouts/layout-types.md:23]: Hi - if it is quicker then that's what we normally do :)

File: src/guides/v2.3/frontend-dev-guide/bk-frontend-dev-guide.md (hunk 1 above)
Comment (DashukVolodymyr) [reply src/guides/v2.3/frontend-dev-guide/bk-frontend-dev-guide.md]: Done!

File: src/guides/v2.3/frontend-dev-guide/bk-frontend-dev-guide.md (hunk 2 above)
Comment (DashukVolodymyr) [reply src/guides/v2.3/frontend-dev-guide/bk-frontend-dev-guide.md]: Done!

File: src/guides/v2.3/frontend-dev-guide/layouts/layout-types.md (hunk 3 above)
Comment (DashukVolodymyr) [reply src/guides/v2.3/frontend-dev-guide/layouts/layout-types.md]: Done!

Comment (DashukVolodymyr) [reply src/guides/v2.3/frontend-dev-guide/layouts/layout-types.md]: @BarnyShergold  all your suggestions have been implemented. Could you please review.

**Review by BarnyShergold — APPROVED** [review]
Excellent!

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
running tests

**Pull request merged**